### DISCUSSION TITLE
Mi serve aiuto per modificare il css

### DISCUSSION THREAD

**Discussion by CobraOfficial01** [iisgiua/giuaschool#765] [Richieste di aiuto]
salve mi potresti creare il registro simile a classeviva grazie

**Comment by robertopadovesi** [comment]
Mi perdoni, ma un registro c'è già. Inoltre, per cortesia ma evitiamo
soprattutto Classeviva, la grafica non è una delle migliori, è ben fruito
ma non è ben organizzato, ne in grafica ne in funzioni.
Poi, non vorrei sembrare scortese, ma non penso che il prof. Dessì, si
metta a fare registri a chiunque, soprattutto gratuitamente e così, al
vento, senza indicazioni o precisazioni.
Un registro non si fa in un click, ma serve tempo e dedizione.
